scripts/security/certs-create-per-user.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+ROOT_CA_DIR=$(realpath $(dirname "${BASH_SOURCE[0]}"))
```

Comment:
@jeanlouisboudart IIUC the goal of this is to provide the real path to this directory, e.g.

```
/Users/yeva/git/cp-demo/scripts/security
```

There is another place in this demo with an example of a command that does this as well, https://github.com/confluentinc/cp-demo/blob/5.5.0-post/scripts/start.sh#L3

```
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```

For consistency, we should use the same command in both places.  Do you have concern if we changed this to:

```
ROOT_CA_DIR ="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```

Reply:
That's exactly the goal.
I'll make that change tomorrow.

Reply:
I don't think we can use this trick here as it change the current directory to that particular folder :).

As such later in the start.sh everything is broken :)

Reply:
is this Ok for you @ybyzek ?

Reply:
@jeanlouisboudart the only outstanding concern of `realpath` (aside from lack of consistency) is that it's non-standard.  Some people have `dirname` but do not have `realpath` (I checked with another user), so it results in a new added dependency to run this demo (e.g. `brew install coreutils` on macos) -- in contrast, I *suspect* (but don't know for sure) that `dirname` is more standard.  Can we try to make `dirname` do what you want it to do?

cc: @rspurgeon for additional ideas.

Reply:
I understand your concerns now @ybyzek and thanks for pointing this.
I removed the dependency and added a helper script doing exactly what we need.